.travis.yml

```diff
@@ -8,14 +8,33 @@ addons:
       - libgconf-2-4
 branches:
   only:
-    - master
+    - travis_ci_deploy
```

Comment:
can we configure `travis_ci_deploy` to be a protected branch on our repo so people can't accidentally push here?